index.html

```diff
@@ -582,26 +582,51 @@ <h3 class="ui header">Indices status and management</h3>
     <pre><code>GET /my_index_name/_upgrade?pretty&amp;human</code></pre>
 
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+	<div class="es2 es5">
+	<a name="reIndexes"></a>
```

Comment:
As we do not add an entry in the menu, this can be removed too.

Reply:
You should remove this option, we don't have enough space in the menu :grimacing: